src/components/LocationPage/ShareButtons.js

```diff
@@ -0,0 +1,135 @@
+import React, { Fragment, useState } from 'react';
+import SocialButtons from './SocialButtons';
+import {
+  SaveOrShareContainer,
+  SaveOrShareButton,
+  DesktopButtonsWrapper,
+  MobileButtonsWrapper,
+  ClickAwayWrapper,
+  SocialButtonsWrapper,
+} from './ShareButtons.style';
+import { ClickAwayListener } from '@material-ui/core';
+import makeChartShareQuote from 'common/utils/makeChartShareQuote';
+import ShareImageUrlJSON from 'assets/data/share_images_url.json';
+
+const InnerContent = props => {
+  const {
+    iconSize,
+    shareURL,
+    shareQuote,
+    county,
+    stateId,
+    countyId,
+    chartIdentifier,
+  } = props;
+
+  const [showShareIcons, setShowShareIcons] = useState(false);
+
+  const hideSocialButtons = () => {
+    const timeoutId = setTimeout(() => {
+      setShowShareIcons(false);
+    }, 1500);
```

Comment:
Why the delay?  Can you add a comment?

Reply:
Yvonne requested that the 'copy link' button changes to say 'copied' when it is clicked. I delayed the dismissal of the social buttons so the user could see that change briefly.
I'll add a comment!